tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -60,9 +76,19 @@ <h2 id='dialog-title'>Waiting for first Session.run() to connect...</h2>
         type: String,
         value: null,
       },
+      _open: {
+        type: Boolean,
+        value: false,
+      },
+      // Suppress the native paper-dialog backdrop. We use a custom one.
+      _useNativeBackdrop: {
```

Comment:
Is there a reason to expose this as a property that's read-only and false by default?  Why not just set `with-backdrop` directly to false?  (This might be a silly question, I'm just curious.)

Reply:
Ah, I had encountered this issue in a prior PR. We can't directly set attribute values to the string "false" in polymer HTML, which is inconvenient. :(
https://stackoverflow.com/a/39855542/795319

I added a comment.